### DISCUSSION TITLE
Don't buy the AI deck until you know dependencies

### DISCUSSION THREAD

**Discussion by prajwas2004** [bitcraze/discussions#1627] [Q&A]
I got this AI deck for $240(not cheap). Nowhere in the purchase do they say that i need jtag cables and Linux to flash this deck to work. 
So i spent another $100 to get the cables and flash(yeah Linux only). Now the stup1d thing won't connect using these cables either. So i can't flash it. 

I have asked for a replacement module and cables which has been flashed and they are giving me feeble tips on getting the cables to work. 
DONT buy this one.  Support and description on dependencies for it is poor. Cables required for this are not provided in the box. 

I have asked for a replacement and i am waiting for a response. If i don't get a response,  I'll just have to report it.

**Comment by gemenerik** [comment]
Hi Praj,

Thank you for sharing your feedback. I can see that the unexpected need for additional components has been frustrating, and I’m sorry for any inconvenience this has caused. As I mentioned in our previous emails, we take your feedback seriously and are working to improve clarity on our store page. If you’re able to share more about how you navigated the purchase information, it would help us ensure these details are more prominent.

As we’ve discussed in our email exchanges, a debugger is required to work with the product, as stated in our documentation and on the store page. This is particularly essential if, for example, the bootloader is accidentally overwritten. We include this information to ensure customers are fully equipped to handle potential troubleshooting scenarios.

If you feel the product isn't a good fit for your needs, we’re happy to arrange a refund. Please email us directly if you’d like to proceed with that option.

**Comment by prajwas2004** [comment]
Rik

AI module is of no use to me currently because I can't flash it.

Multi-ranger deck is throwing 32000 value in all directions and is not
functioning. I had installed this deck above the Crazyflie. I cleaned the
sensors.  I put the deck in the sunniest conditions with invalid values.

I'm a python programmer who works extensively on raspberry pi.

I had hoped that I could use multiple drones of your company for poc and
prototyping to install this module on larger drones.

However, it looks like your products are more for kids and minimal
hobbiests.

Incorrect/Lack of documentation on your website on technical data.  Even
your video tutorials are not up to date.
Your hardware modules are not validated prior to sending to customers.

*You can issue refunds for these two decks and I will ship these back to
you. *

I have received considerable flak for recommending this drone for poc in my
company.

Maybe once your documentation issues and component quality issues are
sorted out,  we can work in the future.

Good luck

Praj
